katcp/inspecting_client.py

```diff
@@ -164,9 +164,6 @@ def __init__(self, host, port, ioloop=None, initial_inspection=None,
                 connected=False, synced=False, model_changed=False, data_synced=False)
         )
 
-    def __del__(self):
```

Comment:
Perhaps document that stop/close needs to be called to ensure cleanup in the class docstring or perhaps in start's docstring?